src/ui/units/datasets/store/reducers/dataset.ts

```diff
@@ -1266,11 +1271,15 @@ export default (state: DatasetReduxState = initialState, action: DatasetReduxAct
         }
         case ADD_AVATAR_PROTOTYPES: {
             const {list, templates: sourceTemplate} = action.payload;
-
+            const isLastPage = list.length <= state.sourcesPagination.limit;
```

Comment:
Why its here? It's a bit unclear that the isFinished attribute is being set in this action